Source/Editor/Windows/SplashScreen.cpp

```diff
@@ -96,45 +87,34 @@ const Char* SplashScreenQuotes[] =
     TEXT("Loading Simulation"),
     TEXT("Get ready for a surprise!"),
     TEXT("Coffee is my fuel"),
-    TEXT("Installing a free copy of Cyberpunk 2077"),
     TEXT("With great power comes great electricity bill"),
     TEXT("Flax was made in the same city as Witcher 3"),
     TEXT("So JavaScript is a scripting version of Java"),
     TEXT("Good things take time.\n~Someone"),
-    TEXT("Get shit done"),
```

Comment:
if u really don't like it grow thicker skin

Reply:
We had the discussion about it here: https://discord.com/channels/437989205315158016/1154620363574083594

Ultimately the guidelines we went on for what should probably stay or go was that anything profane or alluding to real functions and not obviously a joke needed to be filtered out.  I'm going to go ahead and leave this one in the removal because it could harm Flax's ability to make a good impression on educational institutions or enterprise level companies.